tests/Integration/Framework/Test_ProxyManager.cfg

```diff
@@ -0,0 +1,123 @@
+Resources
```

Comment:
Don't call this file "Test_*" (or pytest will try to execute it)

Reply:
removed